README.md

```diff
@@ -141,7 +141,24 @@ There is also an adminsdk json file that contains the secrets that allow admin a
 
 # Deploying the app to App Store and Play Store
 
-We use fastlane lanes for this - [Read the guide here](https://anddigitaltransformation.atlassian.net/wiki/spaces/ML/pages/4487938094/Release+Guide)
+If the git-crypt key has been regenerated/changed, the CI will need to be updated to include this new key.
+We need to upload the key as a base64 encoded string because secure files don't work correctly GitLab CI, to do so:
+
+- Convert it from the binary form via `base64 -i git-crypt-key -o base64-git-crypt-key`
+- Take the contents of the base64 file and update the variable `CRYPT_KEY` in GitLab variables. Make sure it is masked.
```

Comment:
Wait we're using git-crypt again? If so, why not just include all the files in the source and crypt them, rather than just putting in Github ci?

Reply:
Erm no - let me investigate what is going on here